tests/unit/modules/network/sonic/fixtures/sonic_config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+merged_01:
+  module_args:
+    commands: ['snmp-server community public group ro']
```

Comment:
There is no "commands" argument defined for the sonic_config module. What is the purpose of this line and test?

Reply:
Okay. In looking at the module definition, I see that "commands" is an alias for the "lines" argument.

I will resolve this issue.